distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Result_Set.enso

```diff
@@ -67,31 +67,31 @@ type Builder
 
        Arguments:
        - java_builder: The underlying builder object.
-    type Builder_Inferred java_builder
+    Builder_Inferred java_builder
 
     ## PRIVATE
 
        A builder that has an inferred column type at runtime.
 
        Arguments:
        - java_builder: The underlying builder object.
```

Comment:
This comment seems to be copied from `Builder_Inferred` and does not seem matching for the other builder types.